app/components/navbar.hbs

```diff
@@ -0,0 +1,48 @@
+<nav class='nav'>
+  <button type='button' class='hamburger--toggle' {{on 'click' this.toggleNavbar}}>
+    <FaIcon @icon='bars' @size='2x' />
+  </button>
+
+  <ul class='nav__menu {{if this.isNavOpen "active" ""}}'>
+    <li>
+      <a href={{this.HOME_PAGE_URL}} class='nav__home'>
+        <img class='nav__home__img' src='assets/images/rds-logo.png' alt='Real_Dev_Squad' />
+      </a>
+    </li>
+    <li>
+      <a class='nav__home__link nav__element' href={{this.HOME_PAGE_URL}}>
+        Home
+      </a>
+    </li>
+    {{#each this.RDS_APPS_MAPPING as |siteURL|}}
+    <li><a class='nav__element' href={{siteURL.url}}>{{siteURL.siteName}}</a></li>
+    {{/each}}
```

Comment:
Can we add home also from in #each

Reply:
We can but need to extra condition for classes applied. 
I have removed the over simplification of this looping